WordPress/Classes/ViewRelated/System/Notices/NoticePresenter.swift

```diff
@@ -96,21 +104,29 @@ class NoticePresenter: NSObject {
     private func listenToKeyboardEvents() {
         NotificationCenter.default.addObserver(forName: UIResponder.keyboardWillShowNotification, object: nil, queue: nil) { [weak self] (notification) in
             guard let self = self,
-                let currentContainer = self.currentNoticePresentation?.containerView,
                 let userInfo = notification.userInfo,
-                let keyboardFrameValue = userInfo[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue,
-                let durationValue = userInfo[UIResponder.keyboardAnimationDurationUserInfoKey] as? NSNumber else {
+                let keyboardFrameValue = userInfo[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue else {
```

Comment:
Is there a real case where `UIResponder.keyboardFrameEndUserInfoKey` is present but `UIResponder.keyboardAnimationDurationUserInfoKey` is not? Breaking up the guard makes this code a bit harder to reason about, so I want to be sure its necessary.

Reply:
I broke up the `guard` based on **when** the variable is used. In this case, I thought that adding the `durationValue` in here would make it confusing since it's not used until after the second `guard` below. But I guess doing it this way is still confusing so I'll change it so they're together. 😄

Reply:
I could be convinced otherwise if you feel strongly 😉

Reply:
I'm not strongly attached to that decision so I'll change it. 🙂

Reply:
Done in 90140076c9be37a6cf8f3df0cbfcc232da5af9fb.